src/modules/twap/types.ts

```diff
@@ -0,0 +1,23 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+
+export interface TWAPOrder {
+  sellAmount: CurrencyAmount<Token>
+  buyAmount: CurrencyAmount<Token>
+  receiver: string
+  numOfParts: number
+  startTime: number
+  timeInterval: number
+  span: number
+}
+
+export interface TWAPOrderStruct {
+  sellToken: string
+  buyToken: string
+  receiver: string
+  partSellAmount: string
+  minPartLimit: string
+  t0: number
+  n: number
+  t: number
+  span: number
```

Comment:
These parameters are very cryptic. 
Is it because this is coming from the contract?
In that case, can't you import the type from typechain generated types instead?
Otherwise if that's not the case/not possible, can you document what the parameters are?

Reply:
Yeah, it's from smart-contract. Added the link to documentation